File: src/main/java/com/example/onemoretimejava/arrays/MaxItemInArray.java
package com.example.onemoretimejava.arrays;

public class MaxItemInArray {
    public static void main(String[] args) {
        int[] arr={1,2,17,1,5,4};
        maxItem(arr);
    }
    public static void maxItem(int[] arr){
        int maxxx=arr[0];
        for(int i =0;i<arr.length;i++){
           if(arr[i]>maxxx){
               maxxx=arr[i];
           }
        }
        System.out.println(maxxx);
    }
}

File: src/main/java/com/example/onemoretimekotlin/arrays/MaxItemInArray.kt
package com.example.onemoretimekotlin.arrays

fun main(args: Array<String>) {
    val arr = arrayOf(4, 99, 22, 455, 654, 1)
    maxItem(arr)
}

fun maxItem(arrItem: Array<Int>) {
    var maxNum = arrItem[0]
    for (i in 0 until arrItem.size) {
   // for (i in arrItem.indices) {
        if (arrItem[i] > maxNum) {
            maxNum = arrItem[i]
        }

    }
    println(maxNum)
}